src/containers/Tenant/Diagnostics/Overview/TableInfo/TableInfo.tsx

```diff
@@ -0,0 +1,72 @@
+import React, {useMemo} from 'react';
+import cn from 'bem-cn-lite';
+
+import type {TEvDescribeSchemeResult} from '../../../../../types/api/schema';
+
+import {InfoViewer} from '../../../../../components/InfoViewer';
+
+import {getEntityName} from '../../../utils';
+
+import {prepareTableInfo} from './prepareTableInfo';
+
+import './TableInfo.scss';
+
+const b = cn('ydb-diagnostics-table-info');
+
+interface TableInfoProps {
+    data?: TEvDescribeSchemeResult;
+}
+
+export const TableInfo = ({data}: TableInfoProps) => {
+    const entityName = getEntityName(data?.PathDescription);
+
+    const {
+        generalTableInfo = [],
+        tableStatsInfo = [],
+        tabletMetricsInfo = [],
+        partitionConfigInfo = [],
+    } = useMemo(() => prepareTableInfo(data), [data]);
+
+    if (
+        [generalTableInfo, tabletMetricsInfo, partitionConfigInfo, tableStatsInfo.flat()].flat()
+            .length === 0
+    ) {
+        return <div className={b('title')}>{entityName}</div>;
+    }
+
+    return (
+        <div className={b()}>
+            <InfoViewer info={generalTableInfo} title={entityName} className={b('info-block')} />
+            <div className={b('row')}>
+                {tabletMetricsInfo.length > 0 || partitionConfigInfo.length > 0 ? (
+                    <div className={b('col')}>
+                        <InfoViewer
+                            info={tabletMetricsInfo}
+                            title={'Tablet Metrics'}
+                            className={b('info-block')}
+                            nullOnEmptyInfo
+                        />
+                        <InfoViewer
+                            info={partitionConfigInfo}
+                            title={'Partition Config'}
+                            className={b('info-block')}
+                            nullOnEmptyInfo
+                        />
+                    </div>
+                ) : null}
+                <div className={b('col')}>
+                    {tableStatsInfo.map((info, index) => (
+                        <React.Fragment key={index}>
```

Comment:
`React.Fragment` seems to be redundant

Reply:
Explicit `React.Fragment` makes possible to add key prop to it

Reply:
It is valid to add `key` prop directly to `InfoViewer`